scorum/chain.json

```diff
@@ -0,0 +1,120 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "scorum",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "Scorum Network",
+  "chain_id": "mainnet-1",
+  "bech32_prefix": "scorum",
+  "daemon_name": "scorumd",
+  "node_home": "$HOME/.scorum",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "gas",
```

Comment:
This denom must be defined as a base denom in the assetlist

Reply:
fixed